src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,152 @@
 package co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
```

Comment:
Do not use wildcard import. In Intellij settings you can change it not to use wildcard imports